OLD/archive/Thematic/2021/March/progressed/launchdarkly3/content.md

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
its still looks like you not using full-template. 
its making me sad.
again, if there are bugs - feel free to create an issue on markdown-gen repository and explain in details what exactly didn't work and should be fixed